### PR TITLE
please check my table

### PR DESCRIPTION
**Pull request by olyolyoly**

### CODE REVIEW

**Review by sjr-mayden — CHANGES_REQUESTED** [review]
Doesn't currently match portfolio page.

Example1 isn't field name
There isn't currently a description on your page

File: portfolio_2019-03-15.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+# Dump of table portfolio
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `portfolio`;
+
+CREATE TABLE `portfolio` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `Title` mediumtext,
```
Comment (neilbulloch) [inline portfolio_2019-03-15.sql:30]: 1. Do you expect titles to be more than 255 characters? Why not use VARCHAR?
2. Field names are normaly all lowercase.

File: portfolio_2019-03-15.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  `Image URL` mediumtext,
```
Comment (neilbulloch) [inline portfolio_2019-03-15.sql:31]: 1. You could use VARCHAR and set the length to 500 or 1000 characters.
2. Field names are normaly all lowercase.

**Review by neilbulloch — COMMENTED** [review]
reviewed by Neil, have a look at the suggestions :)

**Pull request merged**